x-pack/plugins/lists/server/siem_server_deps.ts

```diff
@@ -7,15 +7,26 @@
 export {
   transformError,
   buildSiemResponse,
-} from '../../siem/server/lib/detection_engine/routes/utils';
+} from '../../siem/server/lib/detection_engine/routes/utils'; // eslint-disable-line @kbn/eslint/no-restricted-paths
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
 export { deleteTemplate } from '../../siem/server/lib/detection_engine/index/delete_template';
```

Comment:
can it be exported from `'../../siem/server`?

Reply:
Yep, we'll do that in a followup, thanks @restrry 👍

Reply:
I will handle this in a follow up PR from my side. Appreciate this fix